gammapy/spectrum/spectrum_analysis.py

```diff
@@ -434,18 +610,30 @@ class SpectralFit(object):
         List of PHA files to fit
     """
 
-    def __init__(self, pha, bkg=None, arf=None, rmf=None):
+    DEFAULT_STAT = 'cash'
+
+    def __init__(self, pha, bkg=None, arf=None, rmf=None, stat=DEFAULT_STAT):
 
         self.pha = [make_path(f) for f in pha]
         self._model = None
         self._thres_lo = None
         self._thres_hi = None
+        self.statistic = stat
```

Comment:
Call it consistently `statistic` or `stat`?
I think Sherpa uses `stat` in most places, no?